artifacthub-repo.yml

```diff
@@ -1,3 +1,4 @@
+repositoryID: 4ff9aa8a-2519-4c63-865b-d4c758de19a3
```

Comment:
Where's this coming from 🤷🏻 ?

Reply:
From artifact hub, it is an ID generated for our repository.